db/structure.sql

```diff
@@ -3,7 +3,7 @@
 --
 
 -- Dumped from database version 11.11 (Debian 11.11-1.pgdg90+1)
--- Dumped by pg_dump version 11.11
+-- Dumped by pg_dump version 11.12 (Debian 11.12-0+deb10u1)
```

Comment:
OOI did you dump this with the docker image `pg_dump` and postgres server or using a brew install postgres server and the client tools (pg_dump etc_?

Reply:
Neither? I bashed into my docker container containing the running app, and ran rake db:migrate. I could be wrong, but this change auto-filled when I raked